script/test-robots

```diff
@@ -109,7 +110,13 @@ def main(args: argparse.Namespace) -> None:
 
         tempdir = Path(tempdir_name)
         # Make a temporary copy of the archives to receive the logs
-        archives = shutil.copytree(args.archives_dir.resolve(), tempdir / 'archives')
+        if args.archives_dir.is_dir():
+            archives = shutil.copytree(args.archives_dir.resolve(), tempdir / 'archives')
+        else:
+            archives = tempdir / 'archives'
+            archives.mkdir()
+            with ZipFile(args.archives_dir) as zipfile:
+                zipfile.extractall(archives)
```

Comment:
While this is easy to do, it's the sort of complexity I would encourage not having. It complicates the implementation (now there's two things to test) as well as for users (now there's two things I need to understand).
I think I'd stick with the original mechanism here -- just working on a directory of zips since that's the format they need to be in anyway to run the matches.